### ISSUE TITLE
validate_ami_bags.py not reporting incorrect image file names 

### ISSUE THREAD

**Issue by rdmarino** [NYPL/ami-tools#53]
validate_ami_bags.py did not catch: NewPictureaf01.JPG

@genfhk

**Comment by nkrabben** [comment]
Sorry for late response. This filename seems weirdly out of spec. Shouldn't it be div_cms_v01_newpicture.jpg?

**Comment by genfhk** [comment]
Yes. Should validate_ami_bags.py have caught this error? It did not.

**Comment by nkrabben** [comment]
Hmmmm... I think it should, but I'll need to look at it more. I read the original issue backwards.

**Comment by genfhk** [comment]
@rdmarino can you locate this file?

**Comment by rdmarino** [comment]
NYPL402123 / 477744

@nkrabben @genfhk
